client/pdo/client/controller/commands/create.py

```diff
@@ -134,17 +137,28 @@ def command_create(state, bindings, pargs) :
     logger.info('Loaded contract code for %s', contract_class)
 
     # ---------- set up the enclave clients ----------
-    try :
+    enclaveclients = []
+    enclave_names = state.get(['Service', 'EnclaveServiceNames'], [])
+    if len(enclave_names) > 0: #use the database to get the list of enclaves for the contract
+        logger.info('Using eservice database to look up service URL for the contract enclave')
+        try:
+            db = ServiceDB_Manager(service_type='eservice', file_name=state.get(['Service', 'EnclaveServiceDatabaseFile']))
+            for name in enclave_names:
+                enclaveclients.append(db.get_serviceclient_by_name(name))
+        except Exception as e:
+            logger.error('Unable to get the eservice clients using the eservice database: ' + str(e)) 
+            sys.exit(-1)   
```

Comment:
there are a couple good reasons not to use string concatenation... in particular... the concatenation happens whether or not the logger needs to display the string. just use standard "printf" style.

and why are you exiting? that's pretty brutal for a cli script. throw an exception.
same for the others

Reply:
thank you. will switch to printf style